packages/flutter_tools/templates/create/android.tmpl/gradlew

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
```

Comment:
How is this file versioned?  Does gradle promise never to change this file?  Do we plan to never change the contents of this file in the template?